Ghidra/Features/Decompiler/src/decompile/cpp/slghscan.cc

```diff
@@ -1374,8 +1374,8 @@ int4 preprocess(int4 cur_state,int4 blank_state)
       expand_preprocmacros(fname);
       slgh->parseFromNewFile(fname);
       fname = slgh->grabCurrentFilePath();
-      sleighin = fopen(fname.c_str(),"r");
-      if (sleighin == (FILE *)0)
+	  errno_t err = fopen_s(&sleighin, fname.c_str(), "r");
```

Comment:
`fopen_s` and `errno_t` aren't available on Gcc/Clang and changes must be made in `slghscan.l` file.

Reply:
Sorry for the mistake, I've revert those changes